runners/samza/src/main/java/org/apache/beam/runners/samza/translation/ConfigContext.java

```diff
@@ -35,12 +34,17 @@ public class ConfigContext {
   private final Map<PValue, String> idMap;
   private AppliedPTransform<?, ?, ?> currentTransform;
   private final SamzaPipelineOptions options;
-  private final Set<String> stateIds;
+  private final Map<String, String> usedStateIdMap;
+  private final Map<String, String> multiParDoStateIdMap;
```

Comment:
Lets rename this for clarity stateIdsToRewrite the value could also be the new state id.

Reply:
Sounds good - made the change.